sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -96,7 +96,7 @@ private[hive] class SparkExecuteStatementOperation(
   private def getNextRowSetInternal(
       order: FetchOrientation,
       maxRowsL: Long): TRowSet = withLocalProperties {
-    log.info(s"Received getNextRowSet request order=${order} and maxRowsL=${maxRowsL} " +
+    log.debug(s"Received getNextRowSet request order=${order} and maxRowsL=${maxRowsL} " +
```

Comment:
hmm... @wangyum @dongjoon-hyun 

should we change to use 

```
    if (log.isDebugEnabled) {
      log.debug( ... )
    }
```

or `logDebug(...)` here?

Using `log.debug` may not avoid the cost of string splicing ？

Reply:
IIUC, this is not a perf critical path, @LuciferYang . In addition, the goal of this PR is not to avoid costs. Instead, it aims UX, `Avoid generating too many logs:`.

Anyway, you can suggest that in general for perf perspective.